src/global/beam/beam.cc

```diff
@@ -22,8 +22,8 @@ extern "C" {
     using namespace eicrecon;
 
     // Divide MCParticle collection based on generator status and PDG
-    std::vector<std::string> outCollections{"BeamElectrons","BeamProtons"};
-    std::vector<std::vector<int>> values{{4,11},{4,2212}};
+    std::vector<std::string> outCollections{"MCBeamElectrons","MCBeamProtons","MCScatteredElectrons","MCScatteredProtons"};
```

Comment:
Should we review addition of MC prefix separately? Do we expect an impact on the existing users?

Reply:
This can be done separately, although there are no existing users as far as I'm aware. The beam directory and beam.cc should probably be renamed to something that encompasses more than just the beam anyway.